### DISCUSSION TITLE
UR Counter isn't updated until HUD is shown

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15560] [Q&A] [answered]
UR Counter will not update its value if HUD was hidden from the beginning of a beatmap, and once the HUD is shown, it displays `0ur` until the next hitobject is hit:
![osu_2021-11-10_20-43-13](https://user-images.githubusercontent.com/53346335/141175353-ae70f539-b93b-4875-bd18-460a03f859a5.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15568
